### DISCUSSION TITLE
Is it possible to color the background cell of a ScatterChart

### DISCUSSION THREAD

**Discussion by SnowPatrolXV** [recharts/recharts#5366] [Q&A]
Hi,

I would like to draw a farmer's diagram ([example](https://scontent-cdg4-1.xx.fbcdn.net/v/t1.6435-9/46083620_248578282487134_2830288649545842688_n.jpg?_nc_cat=102&ccb=1-7&_nc_sid=833d8c&_nc_ohc=JrXZzGyHB_UQ7kNvgGqZi7l&_nc_zt=23&_nc_ht=scontent-cdg4-1.xx&_nc_gid=AYb9QbkDqvnT-y162mZ0XQ1&oh=00_AYCNB6UjWF4veveoT4VnSOo46G1_JSSah3AeztjphtK17w&oe=678AB419)). The scatterChart with cell seems to be the good choice.

But, I am not sure if is it possible to color the background cell of a scatter ? Or maybe to set the background scatter chart with an image ?

**Comment by ckifer** [comment]
You could try something like this https://recharts.org/en-US/examples/BubbleChart

But this isn't what the scatter chart is intended for so it may not be as simple as it seems.

But it is possible to set the `fill` of a scatter item with `Cell`
https://codesandbox.io/p/sandbox/multi-bubble-chart-forked-kj5z3l

See the last row of the chart there
```
  <Scatter data={data01} fill="#8884d8">
          {data01.map((d) => {
            return <Cell fill={d.fill ?? "#111"} />;
          })}
        </Scatter>
```
